src/chains/ethereum_1/examples/transaction.ts

```diff
@@ -336,12 +339,13 @@ const composeERC20MintParams = {
     // await transaction.sign([toEthereumPrivateKey(ropstenPrivate)])
     // console.log('SIG: ', transaction.signatures)
     // console.log(await transaction.send())
-    //
     // ---> Sign and send erc20 mint Transaction
-    // const ropsten = new ChainFactory().create(ChainType.EthereumV1, ropstenEndpoints, {} as ChainSettings)
+    // const ropsten = new ChainFactory().create(ChainType.EthereumV1, ropstenEndpoints, {
+    //   chainForkType: ropstenChainOptions,
+    // } as ChainSettings)
     // await ropsten.connect()
     // // console.log(await ropsten.chainInfo)
-    // const transaction = await ropsten.new.Transaction(ropstenTransactionOptions)
+    // const transaction = await ropsten.new.Transaction({})
```

Comment:
We shouldn't have to provide an empty object - I think the Transaction() constructor's options param should be optional